src/components/DraftSection.vue

```diff
@@ -128,11 +134,95 @@ const heroChosen = (hero: IHero | null) => {
   state.selectedHero = null;
 };
 
+const timerStart = () => {
+  window.clearInterval(timerStore.radiantTimerInterval);
+  window.clearInterval(timerStore.direTimerInterval);
+  timerStore.setTimeByDefault(TimerMapper.RADIANT_MAIN_TIME);
+  timerStore.setTimeByDefault(TimerMapper.DIRE_MAIN_TIME);
+
+  if (state.roundCounter > DEFAULT_NUMBER_OF_ROUNDS) return;
+
+  if (
+    FIRST_PICK_PICK_ROUNDS.includes(state.roundCounter) ||
+    FIRST_PICK_BAN_ROUNDS.includes(state.roundCounter)
+  ) {
+    if (configStore.isRadiantFirstPick) {
+      timerStore.radiantTimerInterval = window.setInterval(() => {
+        if (timerStore.radiantMainTime === 0) {
+          if (timerStore.radiantReserveTime === 0) {
+            heroChosen(null);
```

Comment:
why null?

Reply:
If the player didn't have time to ban hero then I put null against the hero to array of banned heroes in certain position